frontend/packages/dev-console/src/components/topology/components/TopologyGroupResourcesPanel.tsx

```diff
@@ -22,24 +22,19 @@ const TopologyHelmReleaseResourcesPanel: React.SFC<TopologyHelmReleaseResourcesP
     }, [])
     .sort((a, b) => a.localeCompare(b));
 
-  const resourceLists = kinds.reduce((lists, kind) => {
+  return kinds.reduce((lists, kind) => {
     const model = modelFor(kind);
     const resources = manifestResources.filter((resource) => resource.kind === model.kind);
     if (resources.length) {
       lists.push(
         <div key={model.kind}>
           <SidebarSectionHeading text={model.labelPlural} />
-          <TopologyHelmReleaseResourceList
-            resources={resources}
-            releaseNamespace={releaseNamespace}
-          />
+          <TopologyGroupResourceList resources={resources} releaseNamespace={releaseNamespace} />
         </div>,
       );
     }
     return lists;
   }, []);
-
-  return <div className="overview__sidebar-pane-body">{resourceLists}</div>;
```

Comment:
removing the div caused this in the Helm group's sidepanel
![helm-sidepanel](https://user-images.githubusercontent.com/22490998/88334185-03ea0400-cd4f-11ea-8721-0a31c75f8162.png)

Reply:
Had wrapped the wrong component. Updated.
![demo](https://user-images.githubusercontent.com/20013884/88363172-b5fbed00-cd9c-11ea-89ce-6162fcae7a81.png)